src/styles/blocks/page.scss

```diff
@@ -0,0 +1,77 @@
+.page {
+  overflow-x: hidden;
+  font-family: "Helvetica Neue", sans-serif;
+  font-weight: 400;
+  font-size: 14px;
+  color: $c-font-primary;
+
+  scroll-behavior: smooth;
+
+  background-color: $c-background;
+
+  &__body {
+    margin: 0 auto;
+
+    &--with-side-menu {
+      overflow: auto;
+    }
+  }
+
+  &__side-menu {
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    z-index: 10;
```

Comment:
z-index is not the best solution for turning off the scroll, you can use the fix from Miami for that purpose. Also `10`, that's a lot))